openmetadata-ui/src/main/resources/ui/src/utils/Alerts/AlertsUtil.tsx

```diff
@@ -543,7 +550,7 @@ export const getFieldByArgumentType = (
               },
             ]}>
             <AsyncSelect
-              api={getUserOptions}
+              api={getUserBotOptions}
               className="w-full"
```

Comment:
Can we use the function based on the argument type here? since we only need bots to show in case the `argument` value is `updateByUserList`.